src/utils/build.js

```diff
@@ -113,7 +113,7 @@ const forceIncludeDumbPath = (filePath/*, smartPaths*/) => {
   // we include smartPaths just incase you want to check the inclusion of some library
   return (
     (filePath.endsWith('package.json') || filePath.endsWith('definition.json'))
-    || filePath.match(/aws-sdk\/apis\/.*\.json/)
+    || filePath.match(path.sep === '\\' ? /aws-sdk\\apis\\.*\.json/ : /aws-sdk\/apis\/.*\.json/)
```

Comment:
@BrunoBernardino since Travis doesn't support Windows (yet) it's hard to add a test, but I've tested this logic in Parallels:

![](https://cdn.zapier.com/storage/photos/ea3ec7ad9513de77c536b146b0272234.png)